src/screen-components/travel-details-screens/utils.ts

```diff
@@ -246,14 +246,14 @@ export function getFilteredLegsByWalkOrWaitTime(tripPattern: TripPattern) {
 
 export function canSellCollabTicket(tripPattern: TripPattern) {
   const someLegsByTrain = someLegsAreByTrain(tripPattern);
-  const tariffZonesHaveZoneA = (tariffZones?: TariffZone[]) =>
-    tariffZones?.some((a) => a.id === 'ATB:TariffZone:1');
+  const fareZonesHaveZoneA = (fareZones?: FareZone[]) =>
+    fareZones?.some((a) => a.id === 'ATB:FareZone:1');
```

Comment:
Zone A is `ATB:FareZone:10` now

```suggestion
    fareZones?.some((a) => a.id === 'ATB:FareZone:10');
```